.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
There is missing Readme part where you explain steps to run the application and what is needed to run it (what to install first)